lib/src/controls/surfaces/expander.dart

```diff
@@ -213,7 +213,11 @@ class ExpanderState extends State<Expander>
                     ),
                   ),
             ),
-            padding: const EdgeInsetsDirectional.only(start: 16.0),
+            padding: const EdgeInsetsDirectional.only(
+              start: 16.0,
+              top: 16.0,
+              bottom: 16.0,
```

Comment:
Since we now have `minHeight`, this turns out to be unecessary. It's also causing the expander to be extremely tall.

![Fluent UI Showcase for Flutter e mais 1 página - Pessoal — Microsoft​ Edge 13_09_2022 19_12_07](https://user-images.githubusercontent.com/45696119/190018560-8e18dbfa-3457-488a-b11f-40622c78fdb6.png)

Also, I believe it's causing `trailing` and `leading` to be small

Reply:
The problem is in multi-line text. Perhaps top, bottom padding values of 8.0 would be more appropriate? Or I can remove them altogether as you say.

![Frame 1](https://user-images.githubusercontent.com/66072484/190084883-53b1af38-9448-4b66-a1ee-f4798d71a181.jpg)

Reply:
Well, to overcome this issue, I guess the padding must be applied to the header content itself. What do you think?

Reply:
That's actually a much better way to do things. I've Implemented this in the latest commit.